.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.yml]
```

Comment:
Please change to `[*.{yml,yaml}]` to support `yaml` file as well.

Reply:
Done